src/main/java/ch/njol/skript/expressions/ExprSpawn.java

```diff
@@ -38,75 +31,83 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.bukkit.event.world.SpawnChangeEvent;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Spawn")
 @Description("The spawn point of a world.")
-@Examples({"teleport all players to spawn",
-		"set the spawn point of \"world\" to the player's location"})
+@Examples({
+	"teleport all players to spawn",
+	"set the spawn point of \"world\" to the player's location"
+})
 @Since("1.4.2")
 public class ExprSpawn extends PropertyExpression<World, Location> {
+
 	static {
-		Skript.registerExpression(ExprSpawn.class, Location.class, ExpressionType.PROPERTY, "[the] spawn[s] [(point|location)[s]] [of %worlds%]", "%worlds%'[s] spawn[s] [(point|location)[s]]");
+		Skript.registerExpression(ExprSpawn.class, Location.class, ExpressionType.PROPERTY,
+			"[the] spawn[s] [(point|location)[s]] [of %worlds%]",
```

Comment:
2 tabs here for consistency

Reply:
I know our current conventions suggest two tabs of additional space, but I think it may be better if we do one.

Reply:
just for the syntax pattern? things like Java Streams shoukd remain two tabs.